app/lib/features/home/providers/task_providers.dart

```diff
@@ -26,8 +27,8 @@ final myOpenTasksProvider =
             space: SpaceWithProfileData(space, profile),
           ),
         );
+        // }
```

Comment:
is this necessary?
```suggestion
```

Reply:
Just wanted to keep pairing of commented braces.